low-level/framework/rabbitmq/rabbitmq_egress_processor.py

```diff
@@ -308,6 +308,7 @@ def _transmit_msg_on_exchange(self):
 
         except Exception as ex:
             logger.error("RabbitMQegressProcessor, _transmit_msg_on_exchange: %r" % ex)
+            store_queue.put(jsonMsg)
```

Comment:
@Madhura-08  I think instead of this we should change line #299 to:
    except Exception:

Reply:
Ok. But I think those are specific exceptions which we are catching and I think its good to catch specific and the generic one then. So, instead of changing this, I have added generalized catch for that try block and storing the message in consul there also. Please check and let me know if you still see any problem.

Reply:
Keep message string different here from that of line 303

Reply:
@Madhura-08 Not every message to be sent to RMQ queue I think, so please see if adding this generalized catch will create any issues.

Reply:
We may end up filling consul with some unintended messages, like while reading some malformed json, or during json dump or even during legacy iem publish, which we are not bothered at all. So this generalized exception handling, specially store.put() part would be unnecessary.

@sandeepsgt add your points please

Reply:
I think we should not add message to consul for generic exception, because generic exceptions will be anything except rabbitmq connection errors. We only need to send msg if it was not sent because of rabbitmq errors.

Reply:
Thanks Sandeep. Will remove that from the code